synapse/http/site.py

```diff
@@ -82,6 +82,9 @@ def __init__(
         self.reactor = site.reactor
         self._channel = channel  # this is used by the tests
         self.start_time = 0.0
+        self.experimental_cors_msc3886 = getattr(
+            site, "experimental_cors_msc3886", False
+        )
```

Comment:
why getattr here? It appears you've added the attribute to `SynapseSite` so I don't know why we would expect it not to exist

Reply:
Done.